bindata/assets/config/defaultconfig.yaml

```diff
@@ -19,6 +19,8 @@ extendedArguments:
   - "true"
   leader-elect-retry-period:
   - "3s"
+  leader-elect-renew-deadline:
+  - "12s"
```

Comment:
@Rupesh-git-eng can you please comment a short disclaimer here why we do this? Eg. This will make the request timeout 6s and is needed to have a higher value than a 5s DNS timeout.

Or something similar in that sense?

Reply:
This will increase the default timeout from 5s to 6s and that extra one second is sufficient to wait for an answer from the secondary DNS server if the primary DNS server is failed in /etc/resolv.conf of the master node. 

"https://api-int.xxxx.xxx.com:6443/api/v1/namespaces/kube-system/configmaps/kube-controller-manager?timeout=5s"": context deadline exceeded